Automattic/jetpack
# Jetpack Licenses: Dynamic primary button label and link after product activation

Fixes: 1201509629272450-as-1201610923957285

**Changes proposed in this Pull Request:**

This PR implements the following enhancements:

- The primary button after activation should lead the user to the "Recommendations" section if the user has not seen it before.
- Increase the font size of the main heading on licensing activation UI and successful activation UI as well.

**Does this pull request to change what data or activity we track or use?**

No.

**Testing instructions:**

- Set up a new self-hosted site with this branch as the plugin version.
- Activate Jetpack and purchase any available product, make sure you skip the site selection to get the license key.
- Visit wp-admin/admin.php?page=jetpack#/license/activation and activate your product.
- Verify that the title font size is set to 44px, the activation primary button label is "Configure my site", and the link is pointing to the "Recommendations" section.
- Purchase another compatible product with the one you already have, make sure you skip the site selection to get the license key.
- Visit wp-admin/admin.php?page=jetpack#/license/activation and activate your new product.
- Verify that the title font size is set to 44px, the activation primary button label is "View my plans", and the link is pointing to the "My Plan" section.

**Screenshots:**

Activation before visiting "Recommendations" | Activation after visiting "Recommendations"
--- | ---
![Screenshot 2022-01-04 at 14 02 00](https://user-images.githubusercontent.com/1749918/148057833-39173c91-0010-42ae-a772-eb90c4a4f42c.png) | ![Screenshot 2022-01-04 at 14 15 03](https://user-images.githubusercontent.com/1749918/148057860-72dc44ae-1064-463f-8569-ac5a61423faf.png)
